server/text_generation_server/models/vlm_causal_lm.py

```diff
@@ -212,9 +291,10 @@ def batch_tokenized_inputs(
                         processor, image_inputs, config, image_id
                     )
                     image_id += 1
+                elif chunk_type == "video":
+                    full_text += video_text_replacement(processor, video_inputs, config)
```

Comment:
else: raise Error ( we're starting to have many types of chunks)

Reply:
good catch, updated